dev-docs/publisher-api-reference.md

```diff
@@ -938,4 +939,20 @@ The example below shows how to use these methods:
 
 {% endhighlight %}
 
+<a name="module_pbjs.aliasBidder"></a>
+
+### pbjs.aliasBidder(adapterName, aliasedName)
+
+To define an alias for a bidder adapter, call this method at runtime:
+
+{% highlight js %}
+
+pbjs.aliasBidder('appnexusAst', 'newAlias');
+
+{% endhighlight %}
+
+Defining an alias can help avoid user confusion since it's possible to load an adapter with the same string name twice in different contexts (e.g., `"appnexusAst"` may be loaded once by an SSP and once by the publisher).
+
+It's not technically necessary to define an alias, since each copy of an adapter with the same name gets a different ID in the internal bidder registry so Prebid.js can still tell them apart.
+
 </div>
```

Comment:
If you do define an alias and are using `pbjs.sendAllBids`, you will also need to set up additional line items in the ad server that match the name of the alias (e.g `hb_pb_newalias`, `hb_adid_newalias`, `hb_size_newalias`, `hb_deal_newalias`)